packages/lib-classifier/src/plugins/drawingTools/models/tools/Tool/Tool.spec.js

```diff
@@ -184,4 +184,24 @@ describe('Model > DrawingTools > Tool', function () {
       expect(tool.disabled).to.be.true()
     })
   })
+
+  describe('validation', function () {
+    let tool
+    before(function () {
+      tool = Tool.create(toolData)
+      tool.createMark({ id: '1' })
+      tool.createMark({ id: '2' })
+    })
+
+    it('should compute a reduced validity for all marks belonging to the tool', function () {
+      expect(tool.isValid).to.be.true()
+    })
+
+    it('should delete invalid marks', function () {
```

Comment:
Should another (invalid) mark be added to the tool, then before `tool.validate()` there'd be 3 marks and after `tool.validate()` there'd be 2 marks, confirming an invalid mark is deleted on validation?

Reply:
The generic tool's generic mark model defaults to always be valid, so this is technically testing the behavior of the generic tool. Only the transcription line tool has logic for actually invalidating marks, so if we want to test this here, I'd have to rewrite the tests to specifically use the transcription line tool, which would no longer be testing the generic tool. Adding a test to the spec for the transcription line tool makes sense to add (and I missed since I didn't edit that file), so I'll do that.

Reply:
Ok spec added.